### ISSUE TITLE
tail: Support -F option to follow on file removal/truncation

### ISSUE THREAD

**Issue by ADKaster** [SerenityOS/serenity#23097]
We should support this neat feature from GNU Tail (or maybe BSD, I didn't check :^) ).

**Comment by fdellwing** [comment]
Our `FileWatcher` does currently not register a `truncate`. Imho a `truncate` should trigger a `ContentModified` event, but it does not.

**Comment by fdellwing** [comment]
I have solved the `truncation` part.

I'll now look into implementing `--retry` and `--follow=name` (which are both combined in `-F`).

**Comment by tbhaxor** [comment]
@fdellwing Are you still working on this or can I take this issue?

**Comment by tbhaxor** [comment]
> --follow=name

Aren't we taking the file name from the positional arg?

**Comment by fdellwing** [comment]
> It is mode of following. Got it

Yes. It is used in -F to actually keep following a file by name if the original opened file is moved away. By default we keep following the original file as we use the fd.
